packages/docusaurus-plugin-openapi/src/index.ts

```diff
@@ -8,6 +8,7 @@
 import { readFile } from "node:fs/promises";
 import { join, relative, resolve } from "node:path";
 
+import { createMDXLoaderItem } from "@docusaurus/mdx-loader";
```

Comment:
I meant `createMDXLoaderItem` is not officially public API. Do you really need it?

Reply:
Not really, I'll revert it back to the old API. 🙂